### PR TITLE
Extend push_tag event to include tag message and last commit

### PR DESCRIPTION
**Pull request by ayufan**

### CODE REVIEW

**Comment by vsizov** [comment]
@ayufan Can you fix tests?

related https://gitlab.com/gitlab-org/gitlab-ci/issues/90

**Comment by ayufan** [comment]
Updated. Should be ok.

File: app/services/git_tag_push_service.rb [hunk 1]
```diff
@@ -2,22 +2,38 @@ class GitTagPushService
   attr_accessor :project, :user, :push_data
 
   def execute(project, user, oldrev, newrev, ref)
-    @project, @user = project, user
-    
-    @push_data = build_push_data(oldrev, newrev, ref)
+    if Gitlab::Git.tag_ref?(ref)
```
Comment (randx) [inline app/services/git_tag_push_service.rb]: This is tag push service. This code should be executed only on tag push. What sense to check if it is a tag here?

Comment (ayufan) [inline app/services/git_tag_push_service.rb]: I agree. This is not required, but I'm following here GitPushService where such check is done.

Comment (randx) [inline app/services/git_tag_push_service.rb]: @ayufan ah ok. In this case we need to remove check from GitPushService too

Comment (ayufan) [inline app/services/git_tag_push_service.rb]: So should I remove it from here?

Comment (randx) [inline app/services/git_tag_push_service.rb]: yes. ANd from GitPushService too

**Comment by ayufan** [comment]
Updated

**Comment by randx** [comment]
@ayufan looks good!

**Comment by ayufan** [comment]
Thanks

**Pull request merged**